### PR TITLE
Add ADP, bit4coin, Revolut, REWE-ZENTRALFINANZ

### LINKED ISSUE #395: New company suggestion: `ADP, LLC`
The following suggestion was submitted through the website:

```
{
    "categories": [
        "finance"
    ],
    "name": "ADP, LLC",
    "runs": [
        "https://www.workmarket.com/"
    ],
    "web": "https://www.adp.com/",
    "slug": "adpcom",
    "relevant-countries": [
        "all"
    ],
    "sources": [
        "https://github.com/datenanfragen/data/issues/395"
    ]
}
```

**[Edit](https://company-json.netlify.com/#!doc=%7B%22categories%22%3A%5B%22finance%22%5D%2C%22name%22%3A%22ADP%2C%20LLC%22%2C%22runs%22%3A%5B%22https%3A%2F%2Fwww.workmarket.com%2F%22%5D%2C%22web%22%3A%22https%3A%2F%2Fwww.adp.com%2F%22%2C%22slug%22%3A%22adpcom%22%2C%22relevant-countries%22%3A%5B%22all%22%5D%2C%22sources%22%3A%5B%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fissues%2F395%22%5D%7D)**

### LINKED ISSUE #397: New company suggestion: `bit4coin BV`
The following suggestion was submitted through the website:

```
{
    "relevant-countries": [
        "all"
    ],
    "categories": [
        "finance"
    ],
    "name": "bit4coin BV",
    "address": "Johan Huizingalaan 763a\n1066 VH Amsterdam\nNetherlands",
    "email": "support@bit4coin.net",
    "web": "https://bit4coin.net/",
    "sources": [
        "https://bit4coin.net/en/about",
        "https://bit4coin.net/en/privacy",
        "https://github.com/datenanfragen/data/issues/397"
    ],
    "slug": "bit4coinnet"
}
```

**[Edit](https://company-json.netlify.com/#!doc=%7B%22relevant-countries%22%3A%5B%22all%22%5D%2C%22categories%22%3A%5B%22finance%22%5D%2C%22name%22%3A%22bit4coin%20BV%22%2C%22address%22%3A%22Johan%20Huizingalaan%20763a%5Cn1066%20VH%20Amsterdam%5CnNetherlands%22%2C%22email%22%3A%22support%40bit4coin.net%22%2C%22web%22%3A%22https%3A%2F%2Fbit4coin.net%2F%22%2C%22sources%22%3A%5B%22https%3A%2F%2Fbit4coin.net%2Fen%2Fabout%22%2C%22https%3A%2F%2Fbit4coin.net%2Fen%2Fprivacy%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fissues%2F397%22%5D%2C%22slug%22%3A%22bit4coinnet%22%7D)**

### LINKED ISSUE #401: New company suggestion: `Revolut Ltd`
The following suggestion was submitted through the website:

```
{
    "relevant-countries": [
        "all"
    ],
    "categories": [
        "finance"
    ],
    "name": "Revolut Ltd",
    "runs": [
        "Revolut Travel Ltd",
        "Revolut Trading Ltd"
    ],
    "address": "9th Floor 107 Cheapside\nLondon EC2V 6DN\nUnited Kingdom",
    "email": "dpo@revolut.com",
    "web": "https://www.revolut.com",
    "sources": [
        "https://www.revolut.com/en-DE/",
        "https://github.com/datenanfragen/data/issues/401"
    ],
    "suggested-transport-medium": "email",
    "slug": "revolutcom"
}
```

**[Edit](https://company-json.netlify.com/#!doc=%7B%22relevant-countries%22%3A%5B%22all%22%5D%2C%22categories%22%3A%5B%22finance%22%5D%2C%22name%22%3A%22Revolut%20Ltd%22%2C%22runs%22%3A%5B%22Revolut%20Travel%20Ltd%22%2C%22Revolut%20Trading%20Ltd%22%5D%2C%22address%22%3A%229th%20Floor%20107%20Cheapside%5CnLondon%20EC2V%206DN%5CnUnited%20Kingdom%22%2C%22email%22%3A%22dpo%40revolut.com%22%2C%22web%22%3A%22https%3A%2F%2Fwww.revolut.com%22%2C%22sources%22%3A%5B%22https%3A%2F%2Fwww.revolut.com%2Fen-DE%2F%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fissues%2F401%22%5D%2C%22suggested-transport-medium%22%3A%22email%22%2C%22slug%22%3A%22revolutcom%22%7D)**

### PR DESCRIPTION
**Pull request by m-rey**
* Add ADP, LLC (closes #395)
* Add bit4coin BV (closes #397)
* Add Revolut Ltd. (closes #401)
* Add REWE-ZENTRALFINANZ eG

### CODE REVIEW

File: companies/adp-com.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        "ADP Tax Services, Inc."
+    ],
+    "address": "31 Avenue Jules Quentin\n92000 Nanterre\nFrance",
+    "email": "DataProtectionOfficer.ADPEMEA@adp.com",
+    "web": "https://www.adp.com",
+    "sources": [
+        "https://github.com/datenanfragen/data/issues/395",
+        "https://www.adp.com/privacy/pdf/A2CoBDC.pdf",
+        "https://www.adp.com/privacy.aspx",
+        null
```
Comment (baltpeter) [inline companies/adp-com.json]: Probably also a bug in json-forms but one of the sources went missing here.

File: companies/adp-com.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        },
+        {
+            "desc": "Email",
+            "type": "input"
+        }
+    ],
+    "suggested-transport-medium": "email",
+    "comments": [
+        "The \"runs\" list may be incomplete or contain companies that do not operate in the EU."
+    ]
```
Comment (baltpeter) [inline companies/adp-com.json]: I don't think you need to add that comment. :)

Comment (m-rey) [reply companies/adp-com.json]: I've removed the comment.
If you take a look at the PDF linked in the sources, there are more companies that are run by ADP that stated in this JSON entry. However, I think most of them process data of countries where the EU-GDPR doesn't apply.
Should those companies be added, too?

File: companies/adp-com.json (hunk 1 above)
Comment (m-rey) [reply companies/adp-com.json]: Done.

**Comment by baltpeter** [comment]
Oops, closed the wrong issue again. I really need to be more careful, sorry.

File: companies/adp-com.json (hunk 2 above)
Comment (baltpeter) [reply companies/adp-com.json]: I agree, the current list is enough.

**Comment by m-rey** [comment]
No problem. Thanks for adding the relevant countries!

**Pull request merged**